crates/core/liveview-native-core-swift/Tests/LiveViewNativeCoreTests/LiveViewNativeCoreSocketTests.swift

```diff
@@ -1,20 +1,29 @@
 import XCTest
+
 @testable import LiveViewNativeCore
+
 #if canImport(SystemConfiguration)
     import SystemConfiguration
 #endif
 
 let timeout = TimeInterval(30.0)
 
-let connect_url = "http://127.0.0.1:4001/hello";
+let connect_url = "http://127.0.0.1:4001/hello"
 final class LiveViewNativeCoreSocketTests: XCTestCase {
     func testConnect() async throws {
-        let live_socket = try await LiveSocket(connect_url, timeout, "swiftui")
+        let live_socket = try await LiveSocket(connect_url, "swiftui", .none)
```

Comment:
It's a bit out of the scope of this PR but I think you might be able to `extend LiveSocket` in `Support.swift` to have a constructor that only has 2 parameters that calls this constructor with `.none`. Though I don't actually know swift well enough to say for sure.